### PR TITLE
Added requirements.txt and modified README

### PR DESCRIPTION
**Pull request by flashblaze**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+PyGithub
```
Comment (jarodburchill) [inline requirements.txt:1]: This is not required, the batch and bash scripts already do the pip install.

File: README.md [hunk 2]
```diff
@@ -1,79 +1,127 @@
 # ProjectAutomation
+
```
Comment (jarodburchill) [inline README.md:2]: Adding whitespace like this in a README.md doesn't do anything.

File: README.md [hunk 3]
```diff
@@ -1,79 +1,127 @@
...
 ```
 cd ~/ProjectAutomation/mac-linux
 chmod +x new-project
 ```
+
+## Python Packages:
+
+Install the required python packages
+
+`pip install -r requirements.txt`
```
Comment (jarodburchill) [inline README.md:54]: The batch and bash scripts already do this for you.

**Review by jarodburchill — COMMENTED** [review]
I don't feel like anything here is necessary but I appreciate you taking a look at the repo! If you have any other ideas you think this project could benefit from please put in another pull request! Cheers.

File: README.md (hunk 2 above)
Comment (flashblaze) [reply README.md:2]: Markdown plugin for vscode did that. My bad.

Comment (jarodburchill) [reply README.md:2]: No problem at all, thanks for submitting the pull request!

**Pull request closed without merging**